File: greg/day2/valid_anagram.rs
impl Solution {
    pub fn is_anagram(s: String, t: String) -> bool {
        use std::collections::HashMap;
        let mut map = HashMap::new();
        for c in s.chars() {
            let count = map.entry(c).or_insert(0);
            *count += 1;
        }
        for c in t.chars() {
            let count = map.entry(c).or_insert(0);
            *count -= 1;
        }
        for (_, v) in map {
            if v != 0 {
                return false;
            }
        }
        return true;
    }
}

File: jeremie/valid_anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        ss, tt = {}, {}
        for i in s:
            ss[i] = ss.get(i, 0)+1
        for i in t:
            tt[i] = tt.get(i, 0)+1
        return ss == tt